analyses/independent-samples/02-generate-independent-rnaseq.R

```diff
@@ -104,7 +104,8 @@ independent_rna_primary_each <- sample_df %>%
                           independent_level = "each-cohort",
                           histology_df = .,
                           match_type = "independent_dna_plus_only_rna",
-                          tumor_description_rna_only = "primary",seed = 2020) %>%
+                          tumor_description_rna_only = "primary",seed = 2020) %>% 
+  dplyr::arrange(Kids_First_Participant_ID) %>% 
```

Comment:
```suggestion
  dplyr::arrange(Kids_First_Biospecimen_ID) %>% 
```
I forgot to mention, I would use this, since it is unique

Reply:
@jharenza, good point! I forgot `each cohort` might not have unique participant id! :)